skills_ml/algorithms/occupation_classifiers/train.py

```diff
@@ -75,18 +75,25 @@ def train(self, save=True):
                 module_name, class_name = class_path.rsplit(".", 1)
                 module = importlib.import_module(module_name)
                 cls = getattr(module, class_name)
+
                 logging.info(f"training {class_name}")
                 kf = StratifiedKFold(n_splits=self.k_folds, random_state=self.random_state_for_split)
+                model_hash = self._model_hash(self.matrix.metadata, class_name, parameter_config)
+                trained_model_name = class_name.lower() + "_" + model_hash
+                self.storage.path = os.path.join(store_path, score, trained_model_name)
                 if class_name == "SVC" or class_name == "MLPClassifier":
-                    cls_cv = GridSearchCV(cls(), parameter_config, cv=kf, scoring=score)
+                    cls_cv = GridSearchCV(estimator=cls(), param_grid=parameter_config, cv=kf, scoring=score)
```

Comment:
Assigning to the same variable twice in a row looks like a mistake, even though it's intentional. I would just one-line the ProxyObjectWithStorage call, putting each kwarg on its own line for readability